components/EliminatedStatus/EliminatedStatus.tsx

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import React from 'react';
+
+/**
+ * @returns EliminatedStatus component
+ */
+export const EliminatedStatus = (): React.JSX.Element => {
```

Comment:
Does this need a test?

Reply:
I do not believe this needs a test for itself, but I a test in `LeagueCard.test.tsx` for the elimianted state in which this is used.